### PR TITLE
Navbar Bubble Animation 

### LINKED ISSUE #160: Navbar bubble animation
### Feature request-
I would be adding a navbar bubble animation. 

### Is your feature request related to a problem? Please describe. (optional)
N.A

### Describe how this feature will be useful to the users/ contributors-
Users can use this in their portfolio websites as well as other websites which need navigation bars.

#### Are you contributing under any open-source program ?

PSOC
<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

### PR DESCRIPTION
**Pull request by siddhi-244**
## What is the change?
Added navbar bubble animation. 

## Related issue?
closes: #160 

## How was it tested?
It was tested on chrome developer tools.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

-   [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
-   [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/Mirage-UI/pulls) for the same update/change?
-   [x] Have you made corresponding changes to the documentation?
-   [x] Have you tested the code before submission?
-   [] Did you lint and format your code before making the Pull Request?
-   [x] Have you made the GIF for the component you added?

## GIF you created:

https://user-images.githubusercontent.com/69195262/125275764-0ca36380-e32d-11eb-8700-b74a184392f0.mp4

### CODE REVIEW

File: HTML+CSS+JS/Navigation-Bar/Navbar-Bubble-Animation/index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<html>
+<head>
+	<title>Navbar Animation</title>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<nav>
+		<ul>
+			<div class="menu">
```
Comment (riasenpai) [inline HTML+CSS+JS/Navigation-Bar/Navbar-Bubble-Animation/index.html:11]: formatting is a bit off here @Pranav016 ,otherwise rest seems fine

**Review by riasenpai — CHANGES_REQUESTED** [review]

File: HTML+CSS+JS/Navigation-Bar/Navbar-Bubble-Animation/index.html (hunk 1 above)
Comment (Pranav016) [reply HTML+CSS+JS/Navigation-Bar/Navbar-Bubble-Animation/index.html:11]: Thanks for pointing out 👍 
Our prettier bot does some auto formatting when a PR is merged so I think this will be taken care by it.

**Review by Pranav016 — APPROVED** [review]
Looks good 👍
